src/views/RecIM/components/List/index.jsx

```diff
@@ -42,6 +42,28 @@ const ParticipantList = ({
   showInactive,
   callbackFunction,
 }) => {
+  const [newCaptain, setNewCaptain] = useState();
+  const [teamId, setTeamID] = useState();
+
+  useEffect(() => {
+    // find the old capitain to demote
+    participants.map(async (participant) => {
+      if (
+        participants.length > 0 &&
```

Comment:
If you're inside this lambda function, you already know that participants.length is at least 1.
```suggestion
```